15_2_Mining_with_Regtest.md

```diff
@@ -0,0 +1,33 @@
+# 15.2: Mining the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to generate (i.e. mine) blocks using a Regtest (Regression Test) blockchain.
+To generate the Genesis block and the next blocks on a new blockchain requires very minimal proof-of-work, due to the low difficulty and that it follows the Testnet rules.
```

Comment:
The genesis block is not generated, it's hard coded into bitcoin for all networks (main, test, and regtest).